ds0_utils.py

```diff
@@ -75,3 +81,38 @@ def required_commit():
 
     if options and options.forcecommit:
         return options.forcecommit
+
+def repository():
+    options = parse_ec2_userdata()
+
+    if options and options.repository:
+        if ':' in options.repository:
+            (origin, branch) = options.repository.split(':')
+        else:
+            origin = options.repository
+            branch = ''
+        return { 'origin': origin, 'branch': branch }
+    else:
+        return { 'origin': '', 'branch': '' }
+
+def disable_commit_verification():
+    options = parse_ec2_userdata()
+
+    if options and options.disablecommitverification:
+        return options.disablecommitverification
+    else:
+        return False
+
+def allowed_keys():
+    options = parse_ec2_userdata()
+
+    if options and options.allowedkeys:
+        return options.allowedkeys
+    else:
+        default_key = {
+            'id': '7123CDFD',
+            'file': '/home/ubuntu/datastax_ami/repo_keys/DataStax_AMI.7123CDFD.key',
```

Comment:
I have no strong opinion here. It's easier to glance at the directory and know what the keys are for they have human readable names, but easier to reference them unambiguously from code if they're named after their id's.

Unless @joaquincasares has a strong opinions, I'll merge it as is, or if you want to fix the key-names I'm fine with that as well.